terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/views/validation/bytemin.jsp

```diff
@@ -0,0 +1,18 @@
+<h2>@ByteMin Test</h2>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...